flake.nix

```diff
@@ -67,7 +67,11 @@
           inherit (inputs.haskellNix) config;
         };
         inherit (nixpkgs) lib;
-
+        macOS-security =
```

Comment:
You'll need to get someone from devx to look at this

Reply:
This looks sensible. I would however argue that we want to see the error that stack produces if `/usr/bin/security` is not found in the comment as well. This will make it easier to remove this, should it ever be addressable in a better fashion. For nixpkgs built `stack`, this should not happen, [as the relevant packages in nixpkgs are patched to refert to `/usr/bin/security` instead of "just" `security`.](https://github.com/NixOS/nixpkgs/blob/6e39a7fbc02309a6945d87482f8f53816bfb4754/pkgs/development/haskell-modules/configuration-darwin.nix#L111-L122). For Stack provisioned outside of nix. Or really any haskell program that would use `x509-system` or the `crypton` one, this would likely be also an issue if they were built without those patches, *and* used inside a nix-shell that drops /usr/bin from the PATH.

Reply:
Thanks. Added a comment here: https://github.com/IntersectMBO/cardano-cli/pull/895/commits/d0df4c0535cee2076b0081d6d47ff28ced77779a